core/rfb.js

```diff
@@ -1271,6 +1462,42 @@
                 msg_type = this._sock.rQshift8();
             }
 
+            if (this._rfb_atenikvm) {
+                switch (msg_type) {
```

Comment:
Argh. Well that's horribly annoying. Let's hope they never conflict in a way that forces us to choose. :/

Please put a small comment here though explaining that they conflict.